apps/server/src/shared/testing/factory/domainobject/federal-state.do.factory.ts

```diff
@@ -0,0 +1,32 @@
+import { FederalStateDO, FederalStateProps } from '@src/modules/federal-state/domainobject/federal-state.do';
+import { ObjectId } from 'bson';
+import { County } from '@shared/domain';
+import { DomainObjectFactory } from './domain-object.factory';
+
+export const federalStateDoFactory = DomainObjectFactory.define<FederalStateDO, FederalStateProps>(
+	FederalStateDO,
+	() => {
+		return {
+			name: 'Hamburg',
+			id: new ObjectId().toHexString(),
+
+			abbreviation: 'HH',
+			logoUrl:
+				'https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/Coat_of_arms_of_Hamburg.svg/1200px-Coat_of_arms_of_Hamburg.svg.png',
+			counties: [
+				new County({
+					name: 'Hamburg-Mitte',
+					countyId: 2000,
+					antaresKey: '02000',
+				}),
+				new County({
+					name: 'Altona',
+					countyId: 2002,
+					antaresKey: '02002',
+				}),
+			],
+			createdAt: new Date(2020, 1),
+			updatedAt: new Date(2020, 1),
+		};
```

Comment:
I know this is for testing purpose, but shouldn't the factory create unique objects?
if you create 2 federals states, they will have same name, same abbreviation etc, which should be unique